### ISSUE TITLE
Cannot access feeds after upgrade from 5.1.23 to 5.2.0

### ISSUE THREAD

**Issue by rose-a** [Inedo/proget-docker#10]
After upgrading to `5.2.0`, an exception is displayed when accessing the feed page:

![image](https://user-images.githubusercontent.com/15836594/57363268-10516380-7181-11e9-9c86-6cf9cc046281.png)

## Stack trace
```txt
at Npgsql.BackendMessages.RowDescriptionMessage.GetFieldIndex (System.String name) [0x0002d] in <4985dea8374f4819bebb7847d95ad483>:0 
  at Npgsql.NpgsqlDataReader.GetOrdinal (System.String name) [0x0001e] in <4985dea8374f4819bebb7847d95ad483>:0 
  at Inedo.Data.StrongDataReader+<>c__DisplayClass8_0`1[TRow].<Read>b__4 (System.Reflection.PropertyInfo p) [0x0000c] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (TSource[] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x0001c] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x00051] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector) [0x00000] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at Inedo.Data.StrongDataReader+<>c__DisplayClass8_0`1[TRow].<Read>b__0 (System.Type t) [0x0000f] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at System.Collections.Concurrent.ConcurrentDictionary`2[TKey,TValue].GetOrAdd (TKey key, System.Func`2[T,TResult] valueFactory) [0x00034] in <6649516e5b3542319fb262b421af0adb>:0 
  at Inedo.Data.StrongDataReader+<Read>d__8`1[TRow].MoveNext () [0x0007b] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at Inedo.Data.StrongDataReader+<Read>d__6`1[TRow].MoveNext () [0x00095] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Boolean& found) [0x00045] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00000] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at Inedo.ProGet.Data.DB+Context.Feeds_GetFeed (System.Nullable`1[T] Feed_Id, System.String Feed_Name) [0x0003d] in <0a80db718d0047f3abba0661c50ac25c>:0 
  at Inedo.ProGet.Data.DB.Feeds_GetFeed (System.Nullable`1[T] Feed_Id, System.String Feed_Name) [0x00006] in <0a80db718d0047f3abba0661c50ac25c>:0 
  at Inedo.ProGet.WebApplication.Pages.Feeds.BrowseFeedPage.<.ctor>b__3_0 () [0x00014] in <31b6ea8eeef64989995fd38050109514>:0 
  at System.Lazy`1[T].ViaFactory (System.Threading.LazyThreadSafetyMode mode) [0x00043] in <6649516e5b3542319fb262b421af0adb>:0 
  at System.Lazy`1[T].ExecutionAndPublication (System.LazyHelper executionAndPublication, System.Boolean useDefaultConstructor) [0x00022] in <6649516e5b3542319fb262b421af0adb>:0 
  at System.Lazy`1[T].CreateValue () [0x00074] in <6649516e5b3542319fb262b421af0adb>:0 
  at System.Lazy`1[T].get_Value () [0x0000a] in <6649516e5b3542319fb262b421af0adb>:0 
  at Inedo.ProGet.WebApplication.Pages.Feeds.BrowseFeedPage.get_Feed () [0x00000] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.ProGet.WebApplication.Pages.Feeds.BrowseFeedPage.Inedo.ProGet.WebApplication.Security.IScopedByFeed.get_FeedId () [0x00000] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.ProGet.WebApplication.ProGetHttpModule.ValidateSecuredTasks (System.Web.HttpApplication app, System.Web.IHttpHandler handler) [0x0015c] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.ProGet.WebApplication.ProGetHttpModule.ValidatePrivileges (System.Web.HttpApplication app, System.Web.IHttpHandler handler) [0x00012] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.ProGet.WebApplication.ProGetHttpModule.PostMapRequestHandlerAsync (System.Web.HttpApplication app) [0x00161] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.Web.InedoHttpModule.ProcessEnd (System.IAsyncResult ar) [0x00017] in <99b7e11e0aa142a3a029d0f9971efb26>:0 
  at System.Web.HttpApplication.async_callback_completed_cb (System.IAsyncResult ar) [0x0000d] in <3ec20b6a386e451c9090db8d308c8dfe>:0 
```

This cannot be circumvented by switching back to the latest stable! Please help!

**Comment by rose-a** [comment]
## Docker Log

```txt
ProGet version is 5.2.0.73.
Attempting to use shared configuration for ProGet.
Shared configuration file loaded from /usr/share/Inedo/SharedConfig/ProGet.config.
No encryption key is configured. Credentials will be stored in plain text.
Current DB schema version is 5.2.0.73.
Starting the messenger...
Messenger endpoint is tcp://localhost:1000
Reading standard license list...
Importing license list into database...
License import complete.
Initializing Scheduled Job Dispatcher...
Scheduled Job Dispatcher initialized.
Running Scheduled Job Dispatcher...
Checking for scheduled jobs...
Ensuring that all required scheduled jobs are configured...
Unhandled exception: Field not found in row: ReplicationConfiguration_Xml
Initializing Service Heartbeat...
Service Heartbeat initialized.
Running Service Heartbeat...
Service Heartbeat completed.
Initializing Proxy Configuration Monitor...
Proxy Configuration Monitor initialized.
Running Proxy Configuration Monitor...
Proxy Configuration Monitor completed.
Initializing Drop Path Monitor...
Drop Path Monitor initialized.
Running Drop Path Monitor...
Getting list of feeds with a drop path...
Unhandled exception: Field not found in row: ReplicationConfiguration_Xml
Initializing Execution Dispatcher...
Checking for orphaned executions...
Execution Dispatcher initialized.
Running Execution Dispatcher...
Initializing extensions in /var/proget/extensions...
Extensions manager initialization complete.
Checking for executions to dispatch...
Execution Dispatcher completed.
Initializing Connector Health Check...
Connector Health Check initialized.
Running Connector Health Check...
Connector Health Check completed.
Querying https://www.nuget.org/api/v2 for package count...
Querying http://proget.xxxxxxxxxxxxx/nuget/xxx/ for package count...
https://www.nuget.org/api/v2 has 152934 packages.
Connector error: The remote server returned an error: (500) Internal Server Error.
http://proget.xxxxxxxxxxxxx/nuget/xxx/ has unknown package count.
Initializing Feed Replication...
Feed Replication initialized.
Running Feed Replication...
Unhandled exception: Field not found in row: ReplicationConfiguration_Xml
Initializing Docker Upload Cleanup...
Docker Upload Cleanup initialized.
Running Docker Upload Cleanup...
Docker Upload Cleanup completed.
Running Execution Dispatcher...
Checking for executions to dispatch...
Execution Dispatcher completed.
Initializing Connector Cache Check...
Connector Cache Check initialized.
Running Connector Cache Check...
Getting list of connector responses...
No connector responses to check.
Connector Cache Check completed.
Initializing Multipart Upload Cleanup...
Multipart Upload Cleanup initialized.
Running Multipart Upload Cleanup...
Multipart Upload Cleanup completed.

```

**Comment by rose-a** [comment]
Now even on the main page:

![image](https://user-images.githubusercontent.com/15836594/57364144-dda86a80-7182-11e9-85b8-d7d2943081d5.png)

**Exception stack trace:**

```txt
at Npgsql.BackendMessages.RowDescriptionMessage.GetFieldIndex (System.String name) [0x0002d] in <4985dea8374f4819bebb7847d95ad483>:0 
  at Npgsql.NpgsqlDataReader.GetOrdinal (System.String name) [0x0001e] in <4985dea8374f4819bebb7847d95ad483>:0 
  at Inedo.Data.StrongDataReader+<>c__DisplayClass9_0`1[TRow].<ReadAllAsync>b__2 (System.Reflection.PropertyInfo p) [0x0000c] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (TSource[] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x0001c] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x00051] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector, System.Func`2[T,TResult] elementSelector) [0x00000] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at Inedo.Data.StrongDataReader+<>c__DisplayClass9_0`1[TRow].<ReadAllAsync>b__0 (System.Type t) [0x0000f] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at System.Collections.Concurrent.ConcurrentDictionary`2[TKey,TValue].GetOrAdd (TKey key, System.Func`2[T,TResult] valueFactory) [0x00034] in <6649516e5b3542319fb262b421af0adb>:0 
  at Inedo.Data.StrongDataReader.ReadAllAsync[TRow] (Inedo.Data.IDbDataResult dbResult) [0x00203] in <ce29a510be19406b8b569711ec9e0e2c>:0 
  at Inedo.ProGet.Data.DB+Context.Dashboards_GetDashboardInfoAsync (System.Nullable`1[T] Packages_Count, System.Nullable`1[T] IncludeInactive_Indicator) [0x00287] in <0a80db718d0047f3abba0661c50ac25c>:0 
  at Inedo.ProGet.WebApplication.Pages.RootPage.CreateChildControlsAsync () [0x000a2] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.ProGet.WebApplication.Pages.ProGetSimpleAsyncPageBase.InitializeAsync () [0x000c4] in <31b6ea8eeef64989995fd38050109514>:0 
  at Inedo.Web.PageFree.SimplePageBase.ProcessRequestAsync (System.Web.HttpContext context) [0x00084] in <99b7e11e0aa142a3a029d0f9971efb26>:0 
  at System.Web.TaskAsyncResult.Wait (System.IAsyncResult result) [0x00032] in <3ec20b6a386e451c9090db8d308c8dfe>:0 
  at System.Web.HttpTaskAsyncHandler.System.Web.IHttpAsyncHandler.EndProcessRequest (System.IAsyncResult result) [0x00000] in <3ec20b6a386e451c9090db8d308c8dfe>:0 
  at Inedo.Web.Handlers.AsyncHandlerWrapper.EndProcessRequest (System.IAsyncResult result) [0x00000] in <99b7e11e0aa142a3a029d0f9971efb26>:0 
  at System.Web.HttpApplication.async_handler_complete_cb (System.IAsyncResult ar) [0x00015] in <3ec20b6a386e451c9090db8d308c8dfe>:0 
```

**Comment by rose-a** [comment]
![image](https://user-images.githubusercontent.com/15836594/57365336-26612300-7185-11e9-981f-d6e23393b396.png)


Really bad!

**Comment by gdivis** [comment]
Packages should still be there - try `ls -a /var/proget/packages/` We'll get another image pushed soon to fix the exceptions.

**Comment by rose-a** [comment]
yep, I found the packages...

**Comment by gdivis** [comment]
This should all be fixed now if you upgrade to 5.2.2.
